client/db_test.go

```diff
@@ -288,3 +289,46 @@ func TestDebugName(t *testing.T) {
 		return nil
 	})
 }
+
+func TestCommonMethods(t *testing.T) {
+	types := []reflect.Type{
+		reflect.TypeOf(&client.DB{}),
+		reflect.TypeOf(&client.Tx{}),
+		reflect.TypeOf(&client.Batch{}),
+		reflect.TypeOf(client.DB{}.B),
+	}
+	blacklist := map[string]struct{}{
```

Comment:
let's add a sanity check that these methods all exist so this doesn't rot when methods are removed